sdk/notif.go

```diff
@@ -38,6 +46,8 @@ type UserNotificationSettings struct {
 type UserNotificationTemplate struct {
 	Subject string `json:"subject,omitempty" yaml:"subject,omitempty"`
 	Body    string `json:"body,omitempty" yaml:"body,omitempty"`
+	// For VCS
+	DisableComment *bool `json:"disable_comment,omitempty" yaml:"disable_comment,omitempty"`
```

Comment:
Why do we need a pointer ?

Reply:
for the omitempty and as code representation. To not put false when it's false